app/controllers/widgets_controller.rb

```diff
@@ -19,7 +19,7 @@ def record_not_found
   def set_widget
     widget_name = action_name.split('_') - WIDGET_TYPES
     @widget = Object.const_get("#{controller_name.camelize[0..-2]}::#{widget_name.join('_').camelize}")
-                    .new(permitted_params)
+                    .new(permitted_params.to_h.merge(account_id: params[:account_id]))
```

Comment:
This will not fix the original issue. See the reason for this change in the ticket.